ppr-ui/src/utils/mhr-api-helper.ts

```diff
@@ -407,9 +407,10 @@ export async function validateDocumentID (documentId: string) {
   }
 }
 
-export async function submitMhrTransfer (payloadData, mhrNumber) {
+export async function submitMhrTransfer (payloadData, mhrNumber, staffPayment) {
+  const paymentParams = `?${mhrStaffPaymentParameters(staffPayment)}`
   try {
-    const result = await axios.post(`transfers/${mhrNumber}`, payloadData, getDefaultConfig())
+    const result = await axios.post(`transfers/${mhrNumber}${paymentParams}`, payloadData, getDefaultConfig())
```

Comment:
This will always apply the paymentParams but i'm guessing thats intentional? 
Not really an issue but should we check for `staffPayment` perhaps before applying?

Reply:
Intentional yes, would be beneficial to check staffPayment before though. Tested with no payment info and it formats the URL correctly, but I will add a check for safety's sake

Reply:
I guess we do it in the `mhrStaffPaymentParameters` method itself, so i suppose it's ok.